### PR TITLE
MRG: Dont mangle env during setup

### LINKED ISSUE #271: Error on download_version
We are trying to download a legacy version of `expyfun` but getting the following error:

``` python
>>> expyfun.download_version('090948e')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "c:\users\labuser\builds\expyfun\expyfun\_git.py", line 94, in download_version
    assert git_version()[0].lower() == version[:7].lower()
AssertionError
```

When I do `pdb` I get that `git_version()` is `'Unknown'`. Any ideas?

### LINKED ISSUE #305: dismantle __fork__ tracking in setup.py
this was a slightly bad idea at the time, and is no longer necessary.

### PR DESCRIPTION
**Pull request by Eric89GXL**
@drammock on OSX setting the `env` during `setup.py` can break things because it makes `git` no longer available. I think you set at least some of this up -- think it's reasonable?

Closes #271.
Closes #305.

### CODE REVIEW

**Comment by drammock** [comment]
Looks reasonable at first glance. Did you do a `git grep Unknown`? We might test for `fork='Unknown'` somewhere... The download_version code is so hard to test automatically, I wish we had a cleaner way of handling it.

**Comment by Eric89GXL** [comment]
> We might test for fork='Unknown' somewhere...

How does that affect this PR? I don't think I changed that behavior.

**Comment by drammock** [comment]
It looked from the diff like `fork` is no longer getting set as unknown, so that's why I mentioned it. Sorry, I'm probably misunderstanding something... Traveling and viewing on phone makes for bad code reviews.

**Comment by Eric89GXL** [comment]
I just set those variables at the top as the fallback. Actually there were branches in the code where it wasn't set at all, which led to missing variable errors in some cases.

**Comment by drammock** [comment]
Oh, I see that now. I'll be out of town for a week, so if this is urgent you should merge it yourself when it's ready.

**Comment by Eric89GXL** [comment]
Nope, feel free to come back to it later

**Comment by Eric89GXL** [comment]
@drammock should I just get rid of the FORK stuff in this PR?

**Comment by drammock** [comment]
sure, if you're up for it, feel free to kill the fork-detection code here

File: expyfun/tests/test_version.py [hunk 1]
```diff
@@ -25,21 +25,18 @@ def test_version():
     assert_true(all('actual' in str(ww.message) for ww in w))
 
     v = 'b798e65'
```
Comment (drammock) [inline expyfun/tests/test_version.py]: If you change this to a commit hash that happens to start with 0, it reveals a bug on [this line](https://github.com/LABSN/expyfun/pull/303/files#diff-2175524ba23bd6005be929c15c98478eR86)... need to remove the `[0]` in the LHS.  Noticed this by switching `v` here to the commit 090948e that led to #271

File: setup.py [hunk 2]
```diff
@@ -31,45 +31,23 @@ def git_version():
...
         # minimal env; LANGUAGE is used on win32
-        env = dict(LANGUAGE='C', LANG='C', LC_ALL='C')
-        for k in ['SYSTEMROOT', 'PATH']:
-            v = os.environ.get(k)
-            if v is not None:
-                env[k] = v
         return subprocess.Popen(cmd, stdout=subprocess.PIPE,
-                                env=env).communicate()[0]
+                                ).communicate()[0]
+    GIT_REVISION = "Unknown"
```
Comment (drammock) [inline setup.py]: single quotes.

**Comment by Eric89GXL** [comment]
Comments addressed and CIs are happy

**Comment by drammock** [comment]
thanks @Eric89GXL

**Pull request merged**